templates/layouts/configs/_feedback.html

```diff
@@ -1,21 +1,7 @@
 {% do pageConfig | setAttribute("feedback", {
   "id": "feedback",
-    "title": _("Give us feedback and help us improve this service"),
-    "pageTitle": {
-      "value": pageConfig.title
-    },
-    "pageURL": {
-      "value": pageConfig.url
-    },
-    "mailto": {
-      "address": "census2021.feedback@ons.gov.uk",
-      "textBefore": _("Tell us what you think of this service by emailing"),
-      "textAfter": _("If you need help with your census, get in touch with our contact centre."),
-      "subject": _("Census 2021 feedback: ") | urlencode,
-      "body": ("\n\n\n---\n" ~ _("This email is for feedback on the census rehearsal, which will help us to improve Census 2021.\nIf you need help with your census, visit https://www.census.gov.uk/contact-us.")) | urlencode
-    },
-    "button":{
-      "close": {},
-      "send": {}
-    }
+  "header": "What do you think about this service?",
+  "content": "Your comments will help us make improvements",
+  "url": "#0",
+  "linkText": "Give Feedback"
```

Comment:
Also, how can this be tested? I can't see feedback anywhere now.

Reply:
So Ive removed feedback from the base page because the pattern has changed and it will no longer be on every page just on a page at the end of the questionnaire. It also wont be set using pageConfig anymore. I did leave this in because I thought it would be added back in anyway with the feedback PR but I'll remove it for now as it doesn't make sense to be left in

Reply:
The next release of the DS will have the full feedback pattern in it which can then be implemented again in runner